libs/utils/utils.js

```diff
@@ -763,9 +763,9 @@ async function decorateIcons(area, config) {
   if (icons.length === 0) return;
```

Comment:
Should icon loading be moved from area-level to section or block-level to avoid loading icons that might be below the fold? Is that an option?